test/unittest/CMakeLists.txt

```diff
@@ -27,6 +27,12 @@ add_subdirectory(${catch2_SOURCE_DIR} ${catch2_BINARY_DIR})
 set(unittest_sources
     tests/main.cxx
     tests/extent_list.cxx
+    tests/wrappers/block_mods.cpp
```

Comment:
Something to sort out later, but before we get into develop: we should standardise on .cxx or .cpp. I picked cxx since cppsuite uses it, but the S3 project has settled on .cpp.

Reply:
I agree it should be standardised. The [MongoDB server style guide](https://github.com/mongodb/mongo/wiki/Server-Code-Style) uses .cpp, so I think it makes sense to follow that.